### PR TITLE
Feature production

### PR DESCRIPTION
**Pull request by MephistoDevelop**
CR#1

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <% @post.each do |article| %>
+    <tr>
+      <% if logged_in? %>
+        <td><%= article.user.name %></td>
+      <%else%>
+          <td><%= link_to 'Login', login_path %></td>
+      <% end %>
+      <td><%= article.title %></td>
+      <td><%= article.body %></td>
+      <td><%= link_to 'Show', posts_path(article) %></td>
```
Comment (mosaaleb) [inline app/views/posts/index.html.erb]: - In our routes file you don't have show action for post resources, and even if you have, `posts_path(article)` should be `post_path(article)` or just `article` rails is smart enough to know what link will be generated.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<table>
+  <tr>
+    <th>Usuario</th>
+    <th>Title</th>
+    <th>Text</th>
+  </tr>
+
+  <% @post.each do |article| %>
+    <tr>
+      <% if logged_in? %>
```
Comment (mosaaleb) [inline app/views/posts/index.html.erb]: - [ ] It's against recommended to put any logic in the views. Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information.

File: app/models/post.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
```
Comment (mosaaleb) [inline app/models/post.rb]: - [ ] We don't need the foreign key unless you changed the name of the model, like `belongs_to :author, foreign_key: 'user_id'`

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  def new
```
Comment (mosaaleb) [inline app/controllers/posts_controller.rb:6]: - [ ] `PostsController` uses `before_action` to restrict access to the :new and :create methods (uses ApplicationController#signed_in?)
```ruby
before_action :signed_in_only!, only: [:new, :create]

def signed_in_only!
  # signed_in? is defined in ApplicationController
  redirect_to sign_in_path unless signed_in?
end
```

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Excellent work @MephistoDevelop @serdg0 :clap:

#### Most of the requirements are handled yet there are some need some work before moving on.

Please see my comments below

#### Fix these issues and submit a new code review.

#### Happy Coding :computer:

**Review by bellom — APPROVED** [review]
Hi Team,
**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Goodluck in your next project :slightly_smiling_face:_

**Pull request merged**